code/modules/clothing/head/armor.dm

```diff
@@ -574,7 +576,6 @@
 		rad = 50)
 	up = TRUE
 	spawn_blacklisted = TRUE
-	style = STYLE_HIGH
```

Comment:
I've still haven't seen justification for why trash and vendor armor can have HIGH_STYLE, but for paramed armor it's no-no.

Reply:
> trash
Has been changed.

> vendor armor
Can hardly be called armor. It is stylish by design. The paramedic armor offers other advantages too.

Reply:
Paramed is stylish by design as well. Other advantage that you nerfing out of existence in this PR.

Reply:
Orly? Guess what else have same style value? Every `/obj/item/clothing/suit` that is not `/obj/item/clothing/suit/armor`.
Meaning entirety of NT/club/guild/IH/moebuis/technomancer jackets and whatnot. Black paramedic vest have HIGH_STYLE, detective's coat have almost the same stats and style.

Reply:
because they are SUITS with ~~little to~~ no armor, THIS is a VOIDSUIT that already gives you speed.

Reply:
> because they are SUITS with ~little to~ no armor, THIS is a VOIDSUIT that already gives you speed.

Armor on job-related roundstart suits goes up to 30, in case of junk armor and triad up to 40. And this is not a voidsuit, what the hell you're talking about?

Reply:
oh, right, it's not
I still don't like the STYLE_HIGH, at least lower it to STYLE_LOW

Reply:
Because you don't like it? That's it?
That might actually be enough, but then don't forget to change style value for other roundstart suits with armor.